nft-geo-filter

```diff
@@ -298,6 +310,24 @@ class GeoFilter:
                 self.show_subprocess_run_error(err)
                 raise
 
+    def add_policy_logging_rule(self):
+        """Append a final rule with same action of the policy if 
+        counter or logging with match to policy if required"""
+        logging = (self.log_accept and self.policy == "accept") or (self.log_drop and self.policy == "drop")
+        if logging or self.counter == "counter":
+            log_statement = self.create_log_statement(self.policy) if logging else ""
+
```

Comment:
Since `create_log_statement()` is already returning the correct log statement based on the policy and the --log-xxx flags provided, we can just do:

```suggestion
        log_statement = self.create_log_statement(self.policy)
        if log_statement != "" or self.counter == "counter":
```